managing-packages/virtual-environments-and-projects.md

```diff
@@ -144,6 +141,34 @@ Typically, the name of the virtual environment will be at the beginning of each
 (venv) $ project-root-folder
 ```
 
+### Install Packages
+
+To install packages from a file that contains a list of necessary packages/dependencies into the virtual environment, we run the command: 
+
+```bash
+pip install -r requirements.txt
+```
+
+To install a single package into the virtual environment, we run the command:
+
+```bash
+pip install <package_name>
+```
+
+#### Feedback
+
+In the terminal, we should see text that indicates successful installation.
+
+![pip-install](../assets/venv-pip-install.png)
+
+### !callout-info
+
+#### `pip` not `pip3`
+
+When using `pip` to perform a global install we use the command `pip3`. `pip3` will specify to use `pip` with Python 3. When using `pip` inside our virtual environment, we use the command `pip` because our virtual environment is an environment for managing packages with Python 3.
```

Comment:
```suggestion
When performing a global install we use the command `pip3`. `pip3` will specify to use `pip` with Python 3. When using `pip` inside our virtual environment, we use the command `pip` because our virtual environment is an environment for managing packages with Python 3.
```